modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ParameterGroupArgumentResolver.java

```diff
@@ -72,7 +73,7 @@ public T resolve(OperationContext operationContext) {
       }
 
       for (ParameterGroupArgumentResolver<?> childResolver : childResolvers) {
-        childResolver.resolve(operationContext);
+        ((Field) childResolver.getContainer()).set(parameterGroup, childResolver.resolve(operationContext));
```

Comment:
Not glad about this cast but the thing here is that when you are dealing with a "child resolver" (aka nested resolver), you know that the container is going to be a Field (no chance of being a Parameter).